RealmModelGenerator/ViewController.swift

```diff
@@ -43,16 +43,20 @@ class ViewController: NSViewController {
             try! entity.setPrimaryKey(pk)
             
             entity.createAttribute({
-                try! $0.setName("childeName")
+                try! $0.setName("childName")
                 $0.defaultValue = "Bob"
                 $0.hasDefault = true
             })
         }
         
-        user.createRelationship({
-            try! $0.setName("children")
-            $0.destination = child
-        })
+        let relationship = Relationship(name: "children", entity: user)
+        relationship.destination = child
+        user.createRelationship(relationship)
```

Comment:
Is there a reason why you coded it where the Relationship is created outside of the entity as opposed to calling the original create function on the entity and supply the build params?